packages/libs/react-ui/src/components/Footer/Footer.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { Footer } from '@components/Footer';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+describe('Footer', () => {
+  test('renders correctly', () => {
+    const { getByTestId } = render(
+      <Footer.Root variant="web" color="default">
+        <Footer.Panel variant="web">
+          <Footer.LinkItem title={'test'}></Footer.LinkItem>
+        </Footer.Panel>
+      </Footer.Root>,
+    );
+
+    const footerContainer = getByTestId('kda-footer');
+    expect(footerContainer).toBeInTheDocument();
+  });
+});
```

Comment:
Unit testing could be improved to test functionality. eg. having multiple icons or items and making sure they are on visible on screen